src/fparser/statements.py

```diff
@@ -1490,21 +1713,25 @@ def tofortran(self, isfix=None):
             s += ' BIND (%s)' % (', '.join(self.bind))
         return s
 
+
 class Import(StatementWithNamelist):
     """
     IMPORT [ [ :: ] <import-name-list> ]
     """
-    match = re.compile(r'import(\b|\Z)',re.I).match
+    match = re.compile(r'import(\b|\Z)', re.I).match
+
 
 class Forall(Statement):
     """
     FORALL <forall-header> <forall-assignment-stmt>
     <forall-header> = ( <forall-triplet-spec-list> [ , <scalar-mask-expr> ] )
-    <forall-triplet-spec> = <index-name> = <subscript> : <subscript> [ : <stride> ]
+    <forall-triplet-spec> = <index-name>
```

Comment:
I think this should still say `<index-name> = <subscript> : <subscript>....` - perhaps just bring `<index-name>` down to the next line.

Reply:
Moved.